proto/asn1/asn1-universal-types/asn1_universal_types_to_der.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ASN1_UNIVERSAL_TYPES_TO_DER_H_
+#define ASN1_UNIVERSAL_TYPES_TO_DER_H_
+
+#include <vector>
+
+#include "asn1_universal_types.pb.h"
+#include "common.h"
+
+namespace asn1_universal_types {
```

Comment:
We shouldn't need this namespace here, especially when you've got the class.

I know we discussed free functions elsewhere, and so it'd be good to revisit that discussion here, since both the namespace and the struct are mostly acting as 'holders' for free/static functions.

Reply:
I've taken out the name_space and class.